src/libraries/Native/Unix/System.IO.Ports.Native/pal_termios.c

```diff
@@ -418,7 +426,15 @@ int32_t SystemIoPortsNative_TermiosReset(intptr_t handle, int32_t speed, int32_t
         return  -1;
     }
 
+#if HAVE_CFMAKERAW
     cfmakeraw(&term);
+#else
+    term.c_iflag &= ~(IMAXBEL|IGNBRK|BRKINT|PARMRK|ISTRIP|INLCR|IGNCR|ICRNL|IXON);
```

Comment:
On Linux, cfmakeraw shows without the IMAXBEL. It was really just convenience and I'm wondering if we can get away without the fork and always list all options. (that can be done later)

Reply:
Yes. I think three things can be simplified:

* cfmakeraw -> list of options
* cfsetspeed -> i/o speed separately
* statfs -> statvfs
  * former is kernel call, latter is libc standard function, and Solaris/Illumos is known to err to the libc functions generally, without `sysctl(2)` like Linux or `sysctl(3)` like BSD.


I wanted to keep changes simple for the first pass, however, can make these changes and simplify the PR. Wdyt, should we make these three now or postpone it until later?

Reply:
I would do it later. cc: @krwq